api/agentpb/agent.proto

```diff
@@ -93,6 +122,9 @@ message AgentMessage {
     // responses from agent
     Pong pong = 8;
     SetStateResponse set_state = 9;
+    ActionRunResponse action_run_response = 10;
+    ActionCancelResponse action_cancel_response = 11;
+    ActionResult action_result = 12;
```

Comment:
It is not possible to have Response without Request. There should be `CancelActionRequest`/`ActionCancelRequest` from pmm-agent to pmm-managed. The response should be sent from pmm-managed to pmm-agent.